plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_de.properties

```diff
@@ -25,136 +29,175 @@ command.org.eclipse.ui.navigate.collapseAll.name                       = Alle zu
-command.org.jkiss.dbeaver.core.txn.autocommit.name                     = Auto-Commit
-command.org.jkiss.dbeaver.core.txn.log.description                     = Aktuelles Transaktionsprotokoll (ausgef\u00FChrte Abfragen)
-command.org.jkiss.dbeaver.core.txn.log.name                            = Transaktionslog
-command.org.jkiss.dbeaver.core.txn.pending.description                 = Liste aller ausstehenden Transaktionen
-command.org.jkiss.dbeaver.core.txn.pending.name                        = Ausstehenden Transaktionen
-command.org.jkiss.dbeaver.ui.eula.showPopup.description                = Zeige Dialog mit den Bedingungen des Endbenutzer-Lizenzvertrags
-command.org.jkiss.dbeaver.ui.eula.showPopup.name                       = Zeige Bedingungen des Endbenutzer-Lizenzvertrags
-command.org.jkiss.dbeaver.ui.tools.menu.description                    = Kontextmen\u00FC mit Datenbankwerkzeugen anzeigen
-command.org.jkiss.dbeaver.ui.tools.menu.name                           = Kontextwerkzeuge
+command.org.jkiss.dbeaver.core.qm.refresh.description                  = Aktualisieren des Abfrageverlaufs
+                      
 
-content-type.org.jkiss.dbeaver.bookmark.name = Lesezeichen
+command.org.jkiss.dbeaver.core.process.stop.name                       = Prozess beenden
+command.org.jkiss.dbeaver.core.process.stop.description                = Laufenden Prozess beenden
 
-extension-point.org.jkiss.dbeaver.workbenchHandler.name = Workbench-Handler
+command.org.jkiss.dbeaver.core.generate.uuid.name                      = UUID generieren
+command.org.jkiss.dbeaver.core.generate.uuid.description               = Erzeugt UUID/GUID und f\u00FCgt sie in den aktiven Text- oder Dateneditor ein
 
-keyword.org.jkiss.dbeaver.core.keyword.connection                = Datenbankverbindung verbinden
-keyword.org.jkiss.dbeaver.core.keyword.pref.db.dataformat.label  = format date number string locale language
-keyword.org.jkiss.dbeaver.core.keyword.pref.db.general.label     = auto commit cursor rollback resultset lob script
-keyword.org.jkiss.dbeaver.core.keyword.pref.db.qm.label          = query log statement ddl
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.binary.label  = binary hex blob
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.content.label = lob blob clob
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.entity.label  = object entity schema catalog table package column
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.erd.label     = erd diagram
-keyword.org.jkiss.dbeaver.core.keyword.pref.editor.sql.label     = sql script commit fetch timeout
-keyword.org.jkiss.dbeaver.core.keyword.pref.ui.general.label     = update http proxy
 
-maven.respository.central          = Zentrales Repository
-maven.respository.central.unsecure = Zentrales Repository (unsicher)
+menu.navigate     = Navigieren
+menu.database     = &Database
```

Comment:
```suggestion
menu.database     = &Datenbank
```

Reply:
I thought &Database is a placeholder which is replaced by database name at runtime. Is it a normal translation?

Reply:
yes